src/Common/tests/System/Net/Capability.Security.cs

```diff
@@ -2,13 +2,34 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Runtime.InteropServices;
 using System.Security.Cryptography.X509Certificates;
 using System.Threading;
 
 namespace System.Net.Test.Common
 {
     public static partial class Capability
     {
+        // Curl + OSX SecureTransport doesn't support the custom certificate callback.
+        public static bool BackendSupportsCustomCertificateHandling =>
+#if TargetsWindows
```

Comment:
I brought this over from HttpClientEKUTest.cs, so it's not "new code". That being said, I do agree that we should steer away from bad practices, so I'll go the conditional compilation route :)